wp-web-push/wp-web-push-admin.php

```diff
@@ -349,6 +371,48 @@ public function options() {
 </tr>
 </table>
 
+<?php
+  if (USE_VAPID) {
+?>
+<table class="form-table">
+<h2 class="title"><?php _e('Voluntary Application Server Identification (VAPID)', 'web-push'); ?></h2>
+<p><?php _e('VAPID is useful to monitor your push messages. It allows your server to submit information about itself to the push service, which improves application stability, exception handling, and security.', 'web-push'); ?></p>
+
```

Comment:
Could we always show this top of the VAPID table, regardless of whether or not it's supported by the PHP platform? I.e., include just the description and link to VAPID docs with a message like "VAPID requires PHP 5.5+, mcrypt, gmp, and openssl extensions."